### DISCUSSION TITLE
stuck on Resolving channel(s)...

### DISCUSSION THREAD

**Discussion by formernest** [Tyrrrz/DiscordChatExporter#1245] [Q&A]
my dotnet version is 8, and use the latest version on linux x64, stuck on "Resolving channel(s)..."， i use this config works well on maxos

**Comment by Tyrrrz** [comment]
Are you using `--include-threads`? Then it will take a while

**Reply by formernest** [reply]
i have solved it. cause my server need a global https proxy.
